packages/core/core-flows/src/sales-channel/workflows/create-sales-channels.ts

```diff
@@ -1,13 +1,21 @@
 import { CreateSalesChannelDTO, SalesChannelDTO } from "@medusajs/types"
-import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
+import {
+  WorkflowData,
+  WorkflowResponse,
+  createWorkflow,
+} from "@medusajs/workflows-sdk"
 import { createSalesChannelsStep } from "../steps/create-sales-channels"
 
 type WorkflowInput = { salesChannelsData: CreateSalesChannelDTO[] }
 
 export const createSalesChannelsWorkflowId = "create-sales-channels"
 export const createSalesChannelsWorkflow = createWorkflow(
   createSalesChannelsWorkflowId,
-  (input: WorkflowData<WorkflowInput>): WorkflowData<SalesChannelDTO[]> => {
-    return createSalesChannelsStep({ data: input.salesChannelsData })
+  (
+    input: WorkflowData<WorkflowInput>
+  ): WorkflowResponse<WorkflowData<SalesChannelDTO[]>> => {
+    return new WorkflowResponse(
+      createSalesChannelsStep({ data: input.salesChannelsData })
+    )
   }
```

Comment:
This is how all Workflow return values will look like. @adrien2p @olivermrbl @srindom

Reply:
yes I believe so, I would just split it in two steps but no strong opinion
```ts
const salesChannel = createSalesChannelsStep({ data: input.salesChannelsData })
return new WorkflowResponse(salesChannel)
```

Reply:
Lol. I prefer wrapping for simple use-case :). But yeah, no strong opinions either

Reply:
Looping in @shahednasser as well for the docs.